components/Footer.vue

```diff
@@ -51,7 +51,7 @@
 
 			<ContactInfo />
 
-			<SocialMedia />
+			<SocialMedia style="display: none;" />
```

Comment:
The "social media" section has already been hidden via [_footer css](https://github.com/inclusive-design/wecount.inclusivedesign.ca/blob/master/assets/css/layout/_footer.scss#L98-L99).

It seems using inline style might be easier to identify these TODOs. If we decided to use inline style, please remove the extra css mentioned above.

Please also add TODOs for these elements that are temporarily hidden. The same comment applies to https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/152 for hiding "tools" and "our data" in header and https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/153 for hiding tools card.

Reply:
@cindyli I completed this so we can get this merged. @TeddTech FYI.